test/abstractarray.jl

```diff
@@ -775,3 +775,14 @@ end
 #to_shape
 @test Base.to_shape(()) === ()
 @test Base.to_shape(1) === 1
+
+#vec for multiple arrays
```

Comment:
happy to change... should i just replace the `let` with `@testset "Multi-arg vec" begin`?